conductor_api/src/conductor/base.rs

```diff
@@ -222,6 +252,7 @@ impl Conductor {
     }
 
     /// Stop and clear all instances
+    /// @QUESTION: why don't we care about errors on shutdown?
```

Comment:
@lucksus I put this in randomly to remind me to ask: I remember this got changed in a PR, but it feels wrong, why are we ignoring the possibility of shutdown errors?